scss/product/components/_chooser.scss

```diff
@@ -1,58 +1,95 @@
-.tab-chooser {
+.chooser-card {
   display: flex;
   flex-wrap: wrap;
   height: 100%;
   list-style: none;
   padding: 0;
-}
 
-.tab-chooser-item {
-  background-color: $tab-chooser-item-background;
-  border: $border-width solid $gray-100;
-  border-radius: 4px;
-  cursor: pointer;
-  display: block;
-  -webkit-filter: grayscale(1);
-  filter: grayscale(1);
-  margin: 4px;
-  text-align: center;
-  width: $tab-chooser-item-width;
+  .chooser-card-item {
+    background-color: $chooser-card-item-background;
+    border: $border-width solid $gray-100;
+    border-radius: 4px;
+    cursor: pointer;
+    display: block;
+    -webkit-filter: grayscale(1);
+    filter: grayscale(1);
+    margin: 4px;
+    text-align: center;
+    width: $chooser-card-item-width;
+  
+    .chooser-card-head {
+      font-size: 16px;
+      padding: 8px 6px;
+      span{
+        height: 2.5rem;
+        width: 2.5rem;
+        margin:15px 0;
+      }
+      img {
+        height: 2.5rem;
+        width: 2.5rem;
+      }
```

Comment:
Do not style the element directly use the class instead, it was` tab-chooser-logo` for img tag

Reply:
Done